cmd/rule-evaluator/main.go

```diff
@@ -19,6 +19,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/GoogleCloudPlatform/prometheus-engine/internal/pkg/ruleevaluator"
```

Comment:
![image](https://github.com/user-attachments/assets/8e004f47-8589-448a-92b8-192277041f25)

Somehow build does not like this package path. I would be fine on adding this to `cmd/rule-evaluator` as `main` package or under `cmd/rule-evaluator/internal/api.go` etc - keep things simple

Reply:
✅  moved the package to `cmd/rule-evaluator/internal`, which appears to have fixed the issue. Strange though...